src/emmy/calculus/derivative.cljc

```diff
@@ -258,7 +262,7 @@
     (let [tag    (d/fresh-tag)
           lifted (d/bundle-element x 1 tag)]
       (-> (d/with-active-tag tag f [lifted])
-          (d/extract-tangent tag)))))
+          (d/extract-tangent tag ::d/dual)))))
```

Comment:
I should probably bind this and `::tape/tape` to vars so there's no chance to mis-type these.